tests/integration/targets/ec2_transit_gateway/tasks/main.yml

```diff
@@ -15,16 +15,16 @@
         tgw_description: "{{ resource_prefix }}-tgw"
 
     - name: test create transit gateway without tags
```

Comment:
Capitalization of the "name" is mandatory to pass the ansible-lint checks (when we decide to run ansible-lint on tests)